scripts/envhook.py

```diff
@@ -1,17 +1,19 @@
 #! /usr/bin/env python3
 
 import errno
+from importlib.abc import (
```

Comment:
Why is this change in this PR?

Reply:
I was experimenting with PyCharm's changelists, and had your envhook fix saved separately in order to run doctests. When I was rebasing I forgot to "shelve" the envhook fix so it got roped in.